build/config/SignToolData.json

```diff
@@ -127,6 +127,7 @@
         "System.IO.FileSystem.DriveInfo.dll",
         "System.IO.FileSystem.Primitives.dll",
         "System.IO.Pipes.dll",
+        "System.Xml.ReaderWriter.dll",
```

Comment:
See https://github.com/dotnet/roslyn/pull/14879/files#diff-40c048b598b8e56a5af39d5fd18de547 - the portable facade set is generated based on NuGet generated project.lock.json.